src/python/visclaw/data.py

```diff
@@ -621,12 +622,17 @@ def __init__(self, name, figno, fig_type, plotdata):
         self._show = True
         self.add_attribute('plotaxes_dict', {})
         self.add_attribute('type',fig_type)   # = 'each_frame' or 'each_run' or 'each_gauge'
+        self.add_attribute('use_for_kml',False)
+        self.add_attribute('kml_dpi',200)
+        self.add_attribute('kml_xlimits',None)
```

Comment:
I have using the auto for matplotlib, so I would hope there would be a separate set of attributes for the google earth box, instead of getting rid of it.


On Apr 11, 2015, at 2:46 PM, Donna Calhoun <notifications@github.com> wrote:

> In src/python/visclaw/data.py:
> 
> > @@ -621,12 +622,17 @@ def __init__(self, name, figno, fig_type, plotdata):
> >          self._show = True
> >          self.add_attribute('plotaxes_dict', {})
> >          self.add_attribute('type',fig_type)   # = 'each_frame' or 'each_run' or 'each_gauge'
> > +        self.add_attribute('use_for_kml',False)
> > +        self.add_attribute('kml_dpi',200)
> > +        self.add_attribute('kml_xlimits',None)
> I am still reluctant to rely on the plotaxes settings to guess the GE axes limits.
> 
> What would happen if the user sets all axis limits to "auto" (or doesn't set anything). Are there routines in Matplotlib that would allow us to get the guess axes limits? And is it always the case that the user would use the same axis limits for possibly multiple axes per figure? If we guess limits for the wrong set of axes, the user has to recreate all of the figures. Also, it would seem to me that the user would want to be able to specify exactly where GE should place their figure, even if plotaxes is set to 'auto'.
> 
> Maybe one solution is to rename this attribute to something like "kml_latlong_box" to indicate exactly where the frame will be placed in the GE browser.
> 
> —
> Reply to this email directly or view it on GitHub.
>